.eslintrc.json

```diff
@@ -139,7 +139,8 @@
         "forbid": ["for"]
       }
     ],
-    "@typescript-eslint/ban-ts-comment": "off"
+    "@typescript-eslint/ban-ts-comment": "off",
+    "react/react-in-jsx-scope": "off"
```

Comment:
This should have been done a separate PR.

Reply:
Library detection build was failing, that's why it was required here.